solo.json.erb

```diff
@@ -0,0 +1,32 @@
+<%=
+require 'json'
+{
+  :run_list => [ 'recipe[nexus-repository-manager::docker]' ],
+  :java => {
+    :jdk_version => ENV['JAVA_VERSION'],
+    :java_home => ENV['JAVA_HOME'],
+    :install_flavor => 'oracle',
+    :oracle => {
+      :accept_oracle_download_terms => true
+    },
+    :jdk => {
+      '8' => {
```

Comment:
How is this effected if a custom Java Version is supplied? This only handled v8.

Reply:
The java recipe cannot automatically download java 6 or 7 from Oracle. I was able to host the binary on my own server and have it install correctly; however, it appears that Java 1.8 is required.

If Java 1.8 is indeed required, this should probably be removed as an option.

Reply:
Keeping this as an option so the Java url and hash can be customized for Java 8.